roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -65,12 +65,15 @@ def execute_module(module_name=None, module_args=None, tmp=None, task_vars=None,
     If the check can gather logs, tarballs, etc., do so when True; but no need to spend
     the time if they're not wanted (won't be written to output directory).
     """
-
-    def __init__(self, execute_module=None, task_vars=None, tmp=None, want_full_results=False):
+    # pylint: disable=too-many-arguments
+    def __init__(self, execute_module=None, task_vars=None, templar=None, tmp=None,
```

Comment:
Generally it's a good idea to put new arguments at the end to avoid any issues where someone is passing non-keyword arguments but I trust that you've verified the usage.